lms/templates/help_modal.html

```diff
@@ -149,11 +158,106 @@
             $('area,input,select,textarea,button').removeAttr('tabindex');
             $(".help-tab a").focus();
             $leanOverlay.removeAttr('tabindex');
+        },
+        showFeedback = function(event, issue_type, title, subject_label, details_label) {
+            event.preventDefault();
+            DialogTabControls.initializeTabKeyValues("#feedback_form_wrapper", $closeButton);
+            $("#feedback_form input[name='issue_type']").val(issue_type);
+            $("#feedback_form_wrapper header").html("<h2>" + title + "</h2><hr>");
+            $("#feedback_form_wrapper label[data-field='subject']").html(subject_label);
+            $("#feedback_form_wrapper label[data-field='details']").html(details_label);
+            $("#help_wrapper").css("display", "none");
+            $("#feedback_form_wrapper").css("display", "block");
+            if (userAuthenticated && finishedLoadingCourseOptions && courseOptions.length > 1) {
+                $('#feedback_form_course').html(courseOptions.join(''));
+                $('.js_feedback_form_course_select').removeClass('hidden');
+            }
+            $closeButton.focus();
+        },
+        loadCourseOptions = function() {
+            // Prevent race condition that can result from multiple requests being made at the same time.
+            if (courseOptionsLoadInProgress) {
+                return;
+            }
+
+            courseOptionsLoadInProgress = true;
+            $.ajax({
+                url: '/api/enrollment/v1/enrollment',
+                success: function(data) {
+                    var i,
+                        courseDetails,
+                        courseName,
+                        courseId,
+                        option,
+                        defaultOptionText = '${_("- Select -") | n, js_escaped_string}',
```

Comment:
There seems to be two levels of escaping being done in this file, as mako is configured to escape HTML and multiple lines are using `js_escaped_string` in addition to that. I'm not sure whether both are needed or just one.